frontend/talentsearch/public/images/Browse_jobs_graphic_1_desktop_dark.png:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
Oops, I think you didn't mean to commit these?

Reply:
@JoshBeveridge I'm not sure what these are, are they required or can we remove them?

Reply:
I think this is related to Windows and throwing up a warning when you try to open a file downloaded from the Internet.

Reply:
Thanks, removed in [remove zone identifier files](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4293/commits/d054dc0423de56311c864c3027ce4fdb7247c974)

Reply:
Sorry, yes, this happens when you move a file from Windows into WSL using the file explorer.